splunk-otel-android/src/main/java/com/splunk/rum/Config.java

```diff
@@ -155,6 +159,14 @@ public boolean isDiskBufferingEnabled() {
         return diskBufferingEnabled;
     }
 
+    /**
+     * Returns the max number of megabytes that will be used to buffer telemetry data in storage.
+     * If this value is exceeded, older telemetry will be deleted until the usage is reduced.
+     */
```

Comment:
Nit: could you add a similar comment to the builder method? Users of this library don't really have any reason to use the getter methods (I think), they probably only construct the `Config` and never get its properties - and, I suppose, in case of any doubts they'd check the builder javadocs first.